.github/workflows/build-test.yml

```diff
@@ -33,32 +59,79 @@ jobs:
+      run: |
+        git submodule update --init --recursive
+    - name: Run RedBPF build with kernel v4.19 on ubuntu-18.04 container
+      run: |
+        docker run --privileged \
+        -v $PWD:/build \
+        -w /build \
+        $BASE_IMAGE:latest-ubuntu-18.04 \
+        /bin/bash -c 'cargo clean && export KERNEL_VERSION=$(ls --indicator-style=none /lib/modules/) && echo KERNEL_VERSION=$KERNEL_VERSION && cargo build && cargo build --examples'
+
+  fedora-34-build-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build host info
+      run: |
+        uname -a
+        cat /etc/os-release
+    - name: Initialize git submodules
+      run: |
+        git submodule update --init --recursive
+    - name: Run RedBPF build with kernel headers on fedora-34 container
+      # Run 'make prepare' to prevent generated/autoconf.h from being removed
+      # when 'make -qp' is executed by bpf_sys::headers::build_kernel_version
+      run: |
+        docker run --privileged \
+        -v $PWD:/build \
+        -w /build \
+        $BASE_IMAGE:latest-fedora-34 \
+        /bin/bash -c 'make -C /lib/modules/*/build prepare; cargo clean && export KERNEL_VERSION=$(ls --indicator-style=none /lib/modules/) && echo KERNEL_VERSION=$KERNEL_VERSION && cargo build && cargo build --examples'
+    - name: Run RedBPF build with vmlinux on fedora-34 container
+      run: |
+        docker run --privileged \
+        -v $PWD:/build \
+        -w /build \
+        $BASE_IMAGE:latest-fedora-34 \
+        /bin/bash -c 'cargo clean && /lib/modules/*/build/scripts/extract-vmlinux /lib/modules/*/vmlinuz > /boot/vmlinux && export REDBPF_VMLINUX=/boot/vmlinux && cargo build && cargo build --examples'
 
-  fedora-35-build-test:
+  alpine-314-build-test:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: System info
+    - name: Build host info
       run: |
         uname -a
-        lsb_release -a
+        cat /etc/os-release
     - name: Initialize git submodules
       run: |
         git submodule update --init --recursive
-    - name: Run RedBPF build on fedora-35 container
+    - name: Run RedBPF build on alpine-3.14 container
       run: |
         docker run --privileged \
         -v $PWD:/build \
         -w /build \
-        $BASE_IMAGE:latest-fedora-rawhide \
-        /bin/bash -c 'export KERNEL_VERSION=$(ls --indicator-style=none /lib/modules/); echo KERNEL_VERSION=$KERNEL_VERSION; cargo build && cargo build --examples'
+        $BASE_IMAGE:latest-alpine \
+        sh -c 'cargo clean && export KERNEL_VERSION=$(ls --indicator-style=none /lib/modules/) RUSTFLAGS=-Ctarget-feature=-crt-static && echo KERNEL_VERSION=$KERNEL_VERSION && cargo +1.51 build --no-default-features --features llvm11 && cargo +1.51 build --no-default-features --features llvm11 --examples'
 
-  publish: 
+  publish:
     runs-on: ubuntu-latest
     if: startsWith( github.ref, 'refs/tags/v')
     needs:
```

Comment:
Please add all the distros into this list, as this is the list of required checks before a new release.

Reply:
OK